### DISCUSSION TITLE
Trilium desktop on Raspberry Pi?

### DISCUSSION THREAD

**Discussion by Wysardry** [TriliumNext/Trilium#4589] [Q&A]
Is there likely to be a Raspberry Pi version of the Trilium desktop client released at any point?

If not, are there any instructions for building it from source? I couldn't find anything in the wiki.

**Comment by Nriver** [comment]
Short version:
```
npm install
npm run switch-electron
npm run start-electron
```

Should be very similar to step 5 to 10 in how to run on android https://github.com/zadam/trilium/discussions/4542

Step 5: The dependency `base-devel` is specific to the Manjaro system. For other systems, you can refer to the dependency package names in https://github.com/zadam/trilium/blob/master/Dockerfile.

Step 6: Change to downloading the source code at this point.

Step 8: Modify to `npm run switch-electron`.

Step 9: Use `npm run start-electron`.
